handbook/engineering/search/goals.md

```diff
@@ -2,64 +2,52 @@
 
 ## Goals
 
-Goals help us achieve our [vision](index.md#vision) for search.
+### Support large scale organizations
 
-### Mature query language
+Timeline: ~6 months (May 2021)
 
-**Problem:** The Sourcegraph query language does not support the growing complexity of searches being run by our users.
+#### Definition of large scale
 
-Examples:
+Organizations with:
 
-- I want to only show examples from repositories that have been modified in the last 3 months so I see the most recent solutions.
-- I want to find repositories that have not been modified in over a year so I can deprecate them.
-- TODO: add more examples.
+- A Git directory size between 100 GB and 50 TB
+- Total lines of code (LOC) between 1 billion and 5 billion
 
-**Success criteria**
+See [looker](https://sourcegraph.looker.com/looks/436) for current customer data.
```

Comment:
Is there a look which just includes the customers matching the above description? I'd be interested to know how many fit into this category. My main concern is if it is too many, we wouldn't get the benefits of having specific customers in mind when deciding what to work on.

Reply:
Not yet, you would need to toggle Git dir size + LOC in the linked look to get this info. As mentioned in the search sync there's ~10 customers we'd define as large right now. I'll either create a separate look or add a link to a google doc with the names in a follow-up.